### ISSUE TITLE
Compatible with blue button version?

### ISSUE THREAD

**Issue by techmouse** [lukas2511/sky3ds.py#10]
Your software reads, writes and formats perfectly fine. But when I pop the SD card into the Sky3DS, it's unreadable. No amount of tinkering can get the SD card and Sky3DS to cooperate. Then when I pull it out and look at the card using your software, it shows all the information is still on it.

But then I put it into a Windows machine and look at the card using the "official" software, it says the information or format (can't tell exactly what they mean since it's in broken english), is outdated and the card will have to be reformatted. When the card is reformatted and all the information is rewritten, it works just fine in the Sky3DS. This is what makes me wonder if your software isn't compatible with the newer blue button version.

I haven't had the Sky3DS for very long so I'm sorry if I can't be more useful but I really hope I can help fix this since the Windows version is SO buggy. I can only get the Windows version to correctly read, write and delete information about 50% of the time and every time it messes something up I have to start all over again. For example, I asked it to delete ONE rom and it deleted ALL roms, save files and the entire partition.

Short version:
The "official" software says your software writes outdated information or in an outdated format and it's not working on my blue button Sky3DS.

Thank you for your time.

**Comment by lukas2511** [comment]
I was only using this with a sky3ds with blue button, so it should work.

The official software changed the layout a little bit for this weird 32+ game "parking" system, but this didn't change anything on the sky3ds itself, so this code should still work.

You may need to use the integrated format function, or an older version of the official software to initially format the sdcard, not sure if it works if you ever formatted or "converted" using a newer version of the official software.

**Comment by techmouse** [comment]
I wanted to give an update about this. After a LOT of experimentation, I see now the problem was a hardware issue with my Sky3DS. I found the problem and fixed it and now your software works GREAT! I'm no longer afraid to make changes to the SD card and I'm not dependent on Windows anymore.

Thank you so much for your contribution!

While I'm here, I have a quick request. Would it be possible to add a way to backup/restore the save files for ALL games on the SD card, instead of doing each one individually? Or maybe an option to create an image of all data (roms and save files) on the SD card so it would be possible to quickly revert to an earlier setup?

Again, thank you for all of your work!

**Comment by lukas2511** [comment]
There is an "-Z" command line option to the tool that backups all savegames
